internal/jimm/access.go

```diff
@@ -790,18 +790,15 @@ func (j *JIMM) RemoveGroup(ctx context.Context, user *openfga.User, name string)
 }
 
 // ListGroups returns a list of groups known to JIMM.
-func (j *JIMM) ListGroups(ctx context.Context, user *openfga.User, filter pagination.LimitOffsetPagination) ([]dbmodel.GroupEntry, error) {
+// `match` will filter the list for name or uuid matching it.
+func (j *JIMM) ListGroups(ctx context.Context, user *openfga.User, filter pagination.LimitOffsetPagination, match string) ([]dbmodel.GroupEntry, error) {
```

Comment:
could we rename `filter` to `pagination`? otherwise i'm likely to confuse `match` and `filter`.. `match` is used for filtering, but `filter` is used for pagination..

Reply:
yeah, i've address this in the last commit to make it easier to review